### PR TITLE
general.pw.e2e.js

### PR DESCRIPTION
**Pull request by sytolk**

### CODE REVIEW

File: app/reducers/settings-default.ts [hunk 1]
```diff
@@ -152,7 +152,10 @@ export default {
   useTrashCan: true,
   useOCR: false,
   useTextExtraction: false,
-  useGenerateThumbnails: true,
+  useGenerateThumbnails:
```
Comment (uggrock) [inline app/reducers/settings-default.ts:155]: such code is typically in config.js, see https://github.com/tagspaces/tagspaces/blob/develop/app/config.ts#L68

File: app/services/electron-io.ts [hunk 2]
```diff
@@ -712,10 +744,27 @@ export default class ElectronIO {
...
-            saveFile({ ...entry, isNewFile: false, tags: [] }, content);
+          if (entry) {
+            if (entry.isFile && overwrite) {
+              saveFile({ ...entry, isNewFile: false, tags: [] }, content);
+            }
           } else {
-            saveFile({ ...entry, isNewFile: true, tags: [] }, content);
+            saveFile(
+              {
+                name: extractFileName(filePath, AppConfig.dirSeparator),
```
Comment (uggrock) [inline app/services/electron-io.ts:754]: isNewFile is not needed in this case for later use ?

File: app/utils/paths.ts [hunk 3]
```diff
@@ -16,6 +16,7 @@
  *
  */
 
+import pathLib from 'path';
```
Comment (uggrock) [inline app/utils/paths.ts:19]: have you tried this with the web build, I think this was only a nodejs lib...

Comment (sytolk) [reply app/utils/paths.ts:19]: moved from app.ts https://github.com/tagspaces/tagspaces/pull/1481/files#diff-ef10f37f97484a476a875416376a3e86a11d827539909c1cc4a87728d529e824L2110

**Pull request merged**